ui/pages/settings/info-tab/info-tab.component.js

```diff
@@ -113,7 +113,7 @@ export default class InfoTab extends PureComponent {
         <div ref={this.settingsRefs[4]} className="info-tab__link-item">
           <Button
             type="link"
-            href="https://raw.githubusercontent.com/MetaMask/metamask-extension/develop/attribution.txt"
+            href={`https://raw.githubusercontent.com/MetaMask/metamask-extension/v${this.state.version}/attribution.txt`}
```

Comment:
@Gudahtt should handle development vs production env to redirect to develop in first case ?

Reply:
It seems more reasonable to me to always link to the tag. In development and testing, we want this to act as production-like as possible.